assets/reducers/report.js

```diff
@@ -0,0 +1,19 @@
+import { REQUEST_REPORT, RECEIVE_REPORT } from '../actions/report'
+import { morph } from '../utils'
+
+const defaultState = {
+    name: null
```

Comment:
I'm probably just missing it, but I don't see where this state is being used. Can you point me to the right place?